package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "postmodern",
+  "name": "Felfele",
   "version": "0.1.0",
```

Comment:
I think it at this point it would be a good idea to unify the versions as well. Currently on iOS we have 1.0.33, on android 0.9.33 and in the npm it will be present as 0.1.0.

Reply:
great idea :-)